### PR TITLE
minor text updates to sustainability.md and sustainability.yaml

### LINKED ISSUE #489: text updates to IDS sustainability module elements
Some minor updates to the CoST IDS sustainability module elements have been requested. Changes to wording as follows

IDS element | 0.9-dev description | new description
-- | -- | --
Maintenance plan or program | Disclose the maintenance plan or program. | Disclose the maintenance plan or program. **This is the documentation that describes work to prevent the breakdown or malfunctioning of an asset.**
Non-climate co-benefits | Identify potential non-climate impacts that have been factored into the project planning. Select from the list: -economic -social -environmental -gender empowerment | **Disclose and provide further details on the** potential non-climate impacts that have been factored into the project planning. This can comprise, within limitation the following: -economic -social -environmental -gender empowerment **-others (explain)**
Disbursement records | Disbursements dates according to financial agreement versus actual disbursements dates. | **Disclose** disbursements dates according to financial agreement versus actual disbursements dates.
Labour obligations | Disclose labour obligations in the construction contract. This can include, without limitation, the following: -minimum wage -overtime -prohibition of forced labour -prohibition of child labour… | update to US spellings, i.e labour -> labor

### PR DESCRIPTION
**Pull request by odscjen**
**Related issues**

closes #489 I also noticed a few more instances of "labour" rather than "labor" so fixed them too

**Description**

**Merge checklist**

- [ ] [Log your changes](https://ocds-standard-development-handbook.readthedocs.io/en/latest/standard/contributing.html#logging-changes)

Its such minor changes I don't think we need a changelog entry, let me know if you think we do and I can add one

If there are changes to `project-schema.json` or `project-package-schema.json`:

- [ ] Update the examples:
  - [ ] `docs/examples/example.json`
  - [ ] `docs/examples/blank.json`
- [ ] Run `./manage.py pre-commit` to update `docs/_static/i8n.csv`

If you added a new definition to the schema, run `./manage.py pre-commit`.

If you added a new codelist:

  - [ ] Add an entry to `docs/reference/codelists.md`

### CODE REVIEW

File: mapping/sustainability.yaml [hunk 1]
```diff
@@ -381,7 +381,7 @@
     title: Maintenance plan or program
     module: Economic and fiscal
     indicator: Economic viability
-    disclosure format: Disclose the maintenance plan or program.
+    disclosure format: Disclose the maintenance plan or program. This is the documentation that describes work to prevent the breakdown or malfunctioning of an asset.
```
Comment (duncandewhurst) [inline mapping/sustainability.yaml]: Should we wait until https://github.com/open-contracting/infrastructure/issues/237 is resolved and then align the wording here with the definitions of maintenance in the project type and project status codelists?

**Review by duncandewhurst — COMMENTED** [review]
Changes look good apart from the one I commented on.

File: mapping/sustainability.yaml (hunk 1 above)
Comment (odscjen) [reply mapping/sustainability.yaml]: yes that's probably sensible

**Comment by odscjen** [comment]
I attempted to run `.manage.py pre-commit` and got the following error

> File "/***/infrastructure/./manage.py", line 176, in _get_definition_references
    _get_definition_references(value, defn, [*parents, key], project_schema, include_nested)
TypeError: pre_commit.<locals>._get_definition_references() takes from 2 to 4 positional arguments but 5 were given

**Comment by jpmckinney** [comment]
> I attempted to run .manage.py pre-commit and got the following error

Fixed

**Pull request merged**